### PR TITLE
[Testing]: Add NeuroscopeSorting data tests

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Requires a small fix to the allowed metadata schema for the sorting interface to include more general ecephys metadata that might still be desired to be included in historical data where the original raw sources are no longer saved.

### CODE REVIEW

File: src/nwb_conversion_tools/datainterfaces/ecephys/basesortingextractorinterface.py [hunk 1]
```diff
@@ -25,18 +27,41 @@ def get_metadata_schema(self):
...
         metadata_schema["properties"]["Ecephys"]["properties"] = dict(
+            Device=dict(type="array", minItems=1, items={"$ref": "#/properties/Ecephys/properties/definitions/Device"}),
+            ElectrodeGroup=dict(
+                type="array", minItems=1, items={"$ref": "#/properties/Ecephys/properties/definitions/ElectrodeGroup"}
+            ),
+            Electrodes=dict(
+                type="array",
+                minItems=0,
+                renderForm=False,
+                items={"$ref": "#/properties/Ecephys/properties/definitions/Electrodes"},
```
Comment (h-mayorquin) [inline src/nwb_conversion_tools/datainterfaces/ecephys/basesortingextractorinterface.py:42]: What is this attribute `renderForm` for?

**Review by h-mayorquin — APPROVED** [review]
Looks good to me. 
How comes we did not have this attributes in the metadata schema? Were you getting an error without them? I don't understand your comment about historical data and raw sources very well.

**Comment by bendichter** [comment]
@h-mayorquin, @CodyCBakerPhD is saying that even if we don't have the raw data, we might want to know what equipment was used, e.g. the device and electrode group information. He's calling this "historical" because we already have the spike sorted data, so we are unlikely to use the equipment metadata directly. 

`renderForm` is a field used to interact with our metadata form GUI. This tells the form not to render Electrodes. It can only be changed programatically.

**Review by bendichter — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Yup, exactly what Ben said.

That said, I have never once seen a case where there wasn't at least LFP, so take that as you will. Still possibly at least in concept that they didn't even save that file, but did keep the spike trains.

**Pull request merged**